javascript-functions/main.js

```diff
@@ -0,0 +1,35 @@
+function convertMinutestoSecond(minutes) {
+  var seconds = minutes * 60;
+  console.log(seconds);
+}
+convertMinutestoSecond(5);
```

Comment:
This is where you would `console.log` so `console.log('convert function', convertMinutestoSecond(5))`